backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -304,4 +313,117 @@ void null_convertToTribalAffiliation() {
     assertThat(convertToTribalAffiliationExtension((String) null)).isNull();
     assertThat(convertToTribalAffiliationExtension((List<String>) null)).isNull();
   }
+
+  @Test
+  void testOrder_convertToServiceRequest() {
+    var testOrder =
+        new TestOrder(
```

Comment:
nit: it may be worth extracting this `new TestOrder` into a helper method, since you're using it in a few different places.

This is the kind of thing #4462 is intended to address, but that's obviously out of scope here 😄

Reply:
Some general refactor needs to be made with the tests when all of the related fhir work comes together. I think we can make those changes then.